### ISSUE TITLE
scientific calculator

### ISSUE THREAD

**Issue by shubham2704** [codeezzi/utility#7]
#  Create a scientific calculator in html,css and javascript.
- Add navbar in your tool's html file. You can copy the navbar from index.html.
- Add screenshot of your tool in, PR message and also in the folder of your tool
- Before Making the PR Pull the latest changes that have been made to the repository

**Note: Please Don't Make Any Changes In `index.html` AND `contributors.html` **
--

## I have;

- [ ] Read contibuting.md
- [ ] Followed all the guidelines
- [ ] updated readme.md

**Comment by aditimaurya** [comment]
Hello @shubham2704 !! This project sounds cool to me, can you please assign this to me? I'd like to contribute.

**Comment by shubham2704** [comment]
@aditimaurya you still working on this?

**Comment by aditimaurya** [comment]
@shubham2704  yes, yes I'm working on this.  Could you please tell me that is there any deadline to make a PR ?

**Comment by shubham2704** [comment]
@aditimaurya No problem! As our contributing guidelines have been changed so read them before and make changes accordingly before makin the pull request. ✌
- no there is no deadline to make a PR

**Comment by shubham2704** [comment]
@aditimaurya  are you still working on this issue.

**Comment by aditimaurya** [comment]
@shubham2704  yes, I'm working on this and I'll soon submit a PR most probably this weekend or in the upcoming week. I'm having my exams going on, so the schedule is quite hectic I'll soon get back to you. Kindly give some time to me.

**Comment by shubham2704** [comment]
@aditimaurya No problem, there is no time limit to finish any tool it's just we have to confirm regularly from the contributors.
And  `ALL THE BEST FOR EXAMS  👍✌`

**Comment by aditimaurya** [comment]
@shubham2704 I have got a few confusions, so could you please clarify them? As per readme, I've to follow the design of index.html, so could you please clarify what does it mean? Also, do I have to link my tool with index.html or just have to submit the three files containing CSS, js and HTML with the tool's code and linking of those files will be done by you? Please clarify them, I'm actually new to the open-source.

**Comment by shubham2704** [comment]
@aditimaurya 
- Either you have to follow the design theme of index.html or you can create your own design it will be fine.
- No you don't have to link your tool to the index.html. Just submit your files and the linking part will be done by our side.

## If you still have any confusion, feel free to ask before making a PR.
